import/import_script_mock_functions.go

```diff
@@ -0,0 +1,349 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients"
+	"github.com/astronomer/terraform-provider-astro/internal/clients/iam"
+	"github.com/astronomer/terraform-provider-astro/internal/clients/platform"
+	mocksIam "github.com/astronomer/terraform-provider-astro/internal/mocks/iam"
+	mocksPlatform "github.com/astronomer/terraform-provider-astro/internal/mocks/platform"
+	"github.com/samber/lo"
+	"golang.org/x/exp/maps"
+)
+
+func HandleWorkspaces(ctx context.Context, platformClient *mocksPlatform.ClientWithResponsesInterface, iamClient *mocksIam.ClientWithResponsesInterface, organizationId string) (string, error) {
```

Comment:
can all these functions just go in the test file instead?

Reply:
yes, i wanted to separate it so its cleaner but i can move them all in there too